packages/design-system/src/components/Review/Review.jsx

```diff
@@ -25,10 +21,7 @@ export class Review extends React.PureComponent {
       onEditClick,
       editContent,
     } = this.props;
-    const classes = classNames(
-      'ds-c-review ds-u-border-bottom--2 ds-u-padding-y--2 ds-u-justify-content--between ds-u-display--flex',
-      className && className
-    );
+    const classes = classNames('ds-c-review', className && className);
```

Comment:
I'm actually not sure if the check on `className` is necessary. `classNames()` might actually handle that automatically, @nichia can you test and make sure?